pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# What to add
+
+The title and link to a podcast that belongs in our list.
+
+The current categories are:
+- Javascript
+- General Technology & Web Development 
+- Other Technology-Specific 
+- Women in Tech
+
+# How to Contribute
+
+1. Fork the repository
+2. Update the README.md with your podcast suggestion(s)
+    - Append the title and link to the podcast, to the list in the appropriate category
+4. Open a PR directed to our main branch
+5. We will either approve it and merge, or add them manually
+
+- [ ] Title of the podcast is added to the proper category
```

Comment:
I think you should think somewhere along the following example: https://github.com/thisdot/open-source/blob/main/.github/PULL_REQUEST_TEMPLATE.md

see the same template in raw form (notice the comments): https://raw.githubusercontent.com/thisdot/open-source/main/.github/PULL_REQUEST_TEMPLATE.md

the creator of the PR should be able to have a checklist of the expected things. 

Like if they ask the question: Is the podcast linked to the source? --> Yes ---> tick checkbox in the PR. And then we the reviewers can see if it is okay.

I would also add like a check to ensure the PR creator checked the existing data:
`[ ] The podcast is not a duplicate`